packages/react-widgets/__tests__/widgets/utils/sortLayersById.js

```diff
@@ -0,0 +1,72 @@
+import sortLayersById from '../../../src/widgets/utils/sortLayersById';
+
+describe('sortLayersById', () => {
+  let ELEMENTS = [];
+  beforeAll(() => {
+    ELEMENTS = [
+      {
+        id: 'storesLayer0',
+        source: 'storesSource',
+        title: 'Store types 0'
+      },
+      {
+        id: 'storesLayer1',
+        source: 'storesSource',
+        title: 'Store types 1'
+      },
+      {
+        id: 'storesLayer2',
+        source: 'storesSource',
+        title: 'Store types 2'
+      },
+      {
+        id: 'storesLayer3',
+        source: 'storesSource',
+        title: 'Store types 3'
+      }
+    ];
+  });
```

Comment:
This beforeEach shouldn't be necessary after the reference change I suggested in sortLayers

Reply:
is a beforeAll no each, only executed at the beginning

Reply:
Yep! Sorry, but still unnecessary.